docs/reference/overview_of_strategies.rst

```diff
@@ -141,29 +141,29 @@ repository.
    "K68R_", "Fransois Leyvraz", "Not Implemented"
    "K69R_", "Johann Joss", "Not Implemented"
    "K70R_", "Robert Pebly", "Not Implemented"
-   "K71R_", "James E Hill", "Not Implemented"
+   "K71R_", "James E Hall", "Not Implemented"
    "K72R_", "Edward C White Jr", "Not Implemented"
-   "K73R_", "Geroge Zimmerman", "Not Implemented"
+   "K73R_", "George Zimmerman", "Not Implemented"
    "K74R_", "Edward Friedland", "Not Implemented"
    "K74RXX_", "Edward Friedland", "Not Implemented"
-   "K75R_", "P D Harrington", "Not Implemented"
+   "K75R_", "Paul D Harrington", "Not Implemented"
    "K76R_", "David Gladstein", "Not Implemented"
    "K77R_", "Scott Feld", "Not Implemented"
    "K78R_", "Fred Mauk", "Not Implemented"
    "K79R_", "Dennis Ambuehl and Kevin Hickey", Not Implemented
-   "K80R_", "Robyn M Dawes and Mark Batel", Not Implemented
+   "K80R_", "Robyn M Dawes and Mark Batell", Not Implemented
    "K81R_", "Martyn Jones", "Not Implemented"
    "K82R_", "Robert A Leyland", "Not Implemented"
    "K83R_", "Paul E Black", "Not Implemented"
    "K84R_", "T Nicolaus Tideman and Paula Chieruzz", "Not Implemented"
-   "K85R_", "Rober B Falk and James M Langsted", "Not Implemented"
+   "K85R_", "Robert B Falk and James M Langsted", "Not Implemented"
    "K86R_", "Bernard Grofman", "Not Implemented"
    "K87R_", "E E H Schurmann",  "Not Implemented"
    "K88R_", "Scott Appold", "Not Implemented"
    "K89R_", "Gene Snodgrass", "Not Implemented"
    "K90R_", "John Maynard Smith", "Not Implemented"
    "K91R_", "Jonathan Pinkley", "Not Implemented"
-   "K92R_", "Anatol Rapoport", "Not Implemented"
+   "K92R_", "Anatol Rapoport", ":class:`TitForTat <axelrod.strategies.titfortat.TitForTat>`"
```

Comment:
Is there some confusion here (not on your part) with `KTITFORTATC`? I was the one who suggested Rapoport as the author for that (because he is the author of TfT) but perhaps we should change that to be `Unknown` as we don't know who wrote that file <https://github.com/Axelrod-Python/TourExec/blob/v0.2.0/src/strategies/KTitForTatC.f>?

I wonder if they were both run in `TourExec` in Axelrod's second tournament... Perhaps `KTITFORTATC` was not included in the original run but just there as an example? (Just wondering.)